src/xcode/ENA/ENA/Source/Services/Risk/Provider/Model/__tests__/RiskProvidingConfigurationTests.swift

```diff
@@ -75,6 +75,15 @@ final class RiskProvidingConfigurationTests: XCTestCase {
 		XCTAssertEqual(config.nextExposureDetectionDate(lastExposureDetectionDate: future, currentDate: now), now)
 	}
 
+	func testGetNextExposureDetectionDate_Success() {
+		// Test the case where everything just works and you get a valid next date in the future.
+		let now = Date()
+		let nextDate = config.nextExposureDetectionDate(lastExposureDetectionDate: now, currentDate: now)
+		let exposureDetectionInterval: TimeInterval = 24 * 60 * 60
+//		XCTAssertEqual(nextDate, now.addingTimeInterval(exposureDetectionInterval))
```

Comment:
remove commented code + exposureDetectionInterval is not used anymore

Reply:
:white_check_mark: